### PR TITLE
Update the provider-consumer example with a postman collection

### PR DESCRIPTION
**Pull request by agarciagan**
Hi, 
I've added a postman collection to the provider-consumer example. Hope this helps.
Regards, 
Ander

### CODE REVIEW

File: dataspace-connector/provider-consumer/prov_cons.postman_collection [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+		{
+			"key": "agreement",
+			"value": ""
+		},
+		{
+			"key": "dataLink",
+			"value": ""
+		}
+	]
+}
```
Comment (brianjahnke) [inline dataspace-connector/provider-consumer/prov_cons.postman_collection:274]: Missing empty last line

**Review by brianjahnke — CHANGES_REQUESTED** [review]
Hi Ander,

thank you for contributing!
I have requested @sebplorenz to take a look at the PR since he initially provided the example.

File: dataspace-connector/provider-consumer/prov_cons.postman_collection (hunk 1 above)
Comment (agarciagan) [reply dataspace-connector/provider-consumer/prov_cons.postman_collection]: Added

**Review by sebplorenz — CHANGES_REQUESTED** [review]
Hi @agarciagan,
thanks for adding the postman collection. It makes following the steps much easier.
I tested the collection and the last step 'Access Data' fails with 'GET
Error: runtime:extenstions~request: request URL is empty'. Maybe it does not parse the link to the artifact from the previous response correctly.
And could you please add a short description to the README how to run the postman collection as a test.
Thank you very much.

**Comment by agarciagan** [comment]
Hi Sebastian,

We are working on a new version of the example based on the IDS Connector v6. I will update the example, fix the Postman call and push it again after holidays

**Comment by dmichalski** [comment]
Hi @agarciagan !

Hope you had wonderful holidays :)

Can you confirm that the currently committed Postman script should work up to the 'Access Data' step (running against two fresh 6.2.0 instances)? I'm asking because we have some trouble during the contract negotiation and can't figure out whether it's config (wrong rule etc.), wrong request or a bug.

Best Regards

**Comment by agarciagan** [comment]
Hi,

No, the current POSTMAN example does not work with v6. We got to fix it to work with v6, but still have to "clean" the code and update it. Part of the team is still on holidays, I hope we will update the example before 17/09

**Comment by agarciagan** [comment]
Hi Sebastian,
Sorry for the delay. I have tried and it works nicely. However, I get access to the data as gzip. If I create a zip file with the response and open it (for example with 7zip), i get acces to the json weather data. I find it more suitable for the easy working version to remove the accept encoding HTTP header on the last API call to directly receive data as json from postman. I copy the updated postman colection. It is the same code you sent, but with the  accept-encoding header disabled on the last call. The cahnge is

```protocolProfileBehavior": {
				"disabledSystemHeaders": {
					"accept-encoding": true
				}
			

The whole postman collectin is

```{
	"info": {
		"_postman_id": "375eb33a-be27-4094-831a-02f83f835f5e",
		"name": "IDS PUB - SUB",
		"description": "Example from https://github.com/sebplorenz/IDS-Deployment-Examples (branch feature/provider-consumer-example), pending to be merged to https://github.com/International-Data-Spaces-Association/IDS-Deployment-Examples at the IDS-Deployment-Examples/dataspace-connector/provider-consumer/ folder",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "General description",
			"event": [
				{
					"listen": "test",
					"script": {
						"exec": [
							"pm.test(\"Retrieve Catalog ID\", function () {\r",
							"    var jsonData = pm.response.json();\r",
							"    pm.collectionVariables.set(\"catalog\", jsonData[\"ids:resourceCatalog\"][0][\"@id\"])\r",
							"});"
						],
						"type": "text/javascript"
					}
				}
			],
			"request": {
				"method": "POST",
				"header": [],
				"url": {
					"raw": "{{CONSUMER_URL}}/api/ids/description?recipient={{PROVIDER_URL}}",
					"host": [
						"{{CONSUMER_URL}}"
					],
					"path": [
						"api",
						"ids",
						"description"
					],
					"query": [
						{
							"key": "recipient",
							"value": "{{PROVIDER_URL}}"
						}
					]
				}
			},
			"response": []
		},
		{
			"name": "Resource catalog description",
			"event": [
				{
					"listen": "test",
					"script": {
						"exec": [
							"pm.test(\"Retrieve Catalog ID\", function () {\r",
							"    var jsonData = pm.response.json();\r",
							"    pm.collectionVariables.set(\"resource\", jsonData[\"ids:offeredResource\"][0][\"@id\"]);\r",
							"    pm.collectionVariables.set(\"artifact\", jsonData[\"ids:offeredResource\"][0][\"ids:representation\"][0][\"ids:instance\"][0][\"@id\"]);\r",
							"});"
						],
						"type": "text/javascript"
					}
				}
			],
			"request": {
				"method": "POST",
				"header": [],
				"url": {
					"raw": "{{CONSUMER_URL}}/api/ids/description?recipient={{PROVIDER_URL}}&elementId={{catalog}}",
					"host": [
						"{{CONSUMER_URL}}"
					],
					"path": [
						"api",
						"ids",
						"description"
					],
					"query": [
						{
							"key": "recipient",
							"value": "{{PROVIDER_URL}}"
						},
						{
							"key": "elementId",
							"value": "{{catalog}}"
						}
					]
				}
			},
			"response": []
		},
		{
			"name": "Negotiate",
			"event": [
				{
					"listen": "test",
					"script": {
						"exec": [
							"pm.test(\"Retrieve agreement ID\", function () {\r",
							"    var jsonData = pm.response.json();\r",
							"    var agreementLong = jsonData[\"_links\"][\"artifacts\"][\"href\"]\r",
							"    var agreementId = agreementLong.replace(\"http://localhost:8081\", \"\").replace(\"/api/agreements/\", \"\").replace(\"/artifacts{?page,size}\", \"\")\r",
							"pm.environment.get(\"variable_key\");\r",
							"    console.log(agreementLong,agreementId)\r",
							"    pm.collectionVariables.set(\"agreement\", agreementId)\r",
							"});"
						],
						"type": "text/javascript"
					}
				}
			],
			"protocolProfileBehavior": {
				"disabledSystemHeaders": {}
			},
			"request": {
				"method": "POST",
				"header": [],
				"body": {
					"mode": "raw",
					"raw": "[ {\r\n        \"@type\" : \"ids:Permission\",\r\n        \"@id\" : \"http://provider:8080/api/rules/5041c3e5-4933-419d-87d9-474b98ced678\",\r\n        \"ids:description\" : [ {\r\n          \"@value\" : \"provide-access\",\r\n          \"@type\" : \"http://www.w3.org/2001/XMLSchema#string\"\r\n        } ],\r\n        \"ids:title\" : [ {\r\n          \"@value\" : \"Example Usage Policy\",\r\n          \"@type\" : \"http://www.w3.org/2001/XMLSchema#string\"\r\n        } ],\r\n        \"ids:action\" : [ {\r\n          \"@id\" : \"https://w3id.org/idsa/code/USE\"\r\n        }],\r\n        \"ids:target\" : \"{{artifact}}\"\r\n} ]",
					"options": {
						"raw": {
							"language": "json"
						}
					}
				},
				"url": {
					"raw": "{{CONSUMER_URL}}/api/ids/contract?recipient={{PROVIDER_URL}}&resourceIds={{resource}}&artifactIds={{artifact}}&download=false",
					"host": [
						"{{CONSUMER_URL}}"
					],
					"path": [
						"api",
						"ids",
						"contract"
					],
					"query": [
						{
							"key": "recipient",
							"value": "{{PROVIDER_URL}}"
						},
						{
							"key": "resourceIds",
							"value": "{{resource}}"
						},
						{
							"key": "artifactIds",
							"value": "{{artifact}}"
						},
						{
							"key": "download",
							"value": "false"
						}
					]
				}
			},
			"response": []
		},
		{
			"name": "Consume",
			"event": [
				{
					"listen": "test",
					"script": {
						"exec": [
							"pm.test(\"Retrieve data URL\", function () {\r",
							"    var jsonData = pm.response.json();\r",
							"    var dataLink = jsonData[\"_embedded\"][\"artifacts\"][0][\"_links\"][\"data\"][\"href\"]\r",
							"    pm.collectionVariables.set(\"dataLink\", dataLink)\r",
							"});"
						],
						"type": "text/javascript"
					}
				}
			],
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "{{CONSUMER_URL}}/api/agreements/{{agreement}}/artifacts",
					"host": [
						"{{CONSUMER_URL}}"
					],
					"path": [
						"api",
						"agreements",
						"{{agreement}}",
						"artifacts"
					]
				}
			},
			"response": []
		},
		{
			"name": "Access Data",
			"protocolProfileBehavior": {
				"disabledSystemHeaders": {
					"accept-encoding": true
				}
			},
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "{{dataLink}}",
					"host": [
						"{{dataLink}}"
					]
				}
			},
			"response": []
		}
	],
	"auth": {
		"type": "basic",
		"basic": [
			{
				"key": "password",
				"value": "password",
				"type": "string"
			},
			{
				"key": "username",
				"value": "admin",
				"type": "string"
			}
		]
	},
	"event": [
		{
			"listen": "prerequest",
			"script": {
				"type": "text/javascript",
				"exec": [
					""
				]
			}
		},
		{
			"listen": "test",
			"script": {
				"type": "text/javascript",
				"exec": [
					""
				]
			}
		}
	],
	"variable": [
		{
			"key": "CONSUMER_URL",
			"value": "http://localhost:8081"
		},
		{
			"key": "PROVIDER_URL",
			"value": "http://provider:8080/api/ids/data"
		},
		{
			"key": "catalog",
			"value": ""
		},
		{
			"key": "resource",
			"value": ""
		},
		{
			"key": "artifact",
			"value": ""
		},
		{
			"key": "agreement",
			"value": ""
		},
		{
			"key": "dataLink",
			"value": ""
		}
	]
}

**Pull request closed without merging**